### PR TITLE
Saved Search Widget

### PR DESCRIPTION
**Pull request by spyderman4g63**
Created a basic widget that saves the current search results page to the current_user.properties in Zookeeper. We could remove this item if that functionality was added to the core product.

### CODE REVIEW

**Comment by spyderman4g63** [comment]
Sorry, first pull request. I forgot to add the actual widget file. Can I update the pull request or do a submit a new one?

**Comment by spyderman4g63** [comment]
I added committed the file. Looks like it's in the request now.

File: save_search_page/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Save Current Search Results Page widget #
+
+# Description and Use #
+
+This widget is designed to be used on the search results page within Watson Explorer Application Builder.
+
+The intention is to save the current url to the current_user.properties in zookeeper. We create a new store called current_user.properties["saved_searches"]. This property accepts a key/value pair. The key is a lable defined by the user and the value is always the applications current url. Currently if the user enters a key that already exists the current entry is updated.
```
Comment (jeanlange) [inline save_search_page/README.md]: save the current url => save the current URL (capitalization)
The key is a lable => The key is a label (spelling)
applications current url => application's current URL (apostrophe & capitalization)
Currently if the user enters a key that... => If the user enters a key that... (remove 'currently')
that already exists the current => that already exists, the current (add comma)

File: save_search_page/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+We save the current relative url and params. This allows the user to select refinements, pages, etc and save the exact state of the page they are on.
```
Comment (jeanlange) [inline save_search_page/README.md]: url => URL

**Comment by jetulis** [comment]
Please add a note saying that Widget needs to be in no-async mode to save search url correctly. If async is on, ajax-call url will be saved into zookeeper.

**Comment by spyderman4g63** [comment]
Submitted pull request to address that issue.

**Pull request merged**